pkg/restore/restore.go

```diff
@@ -627,6 +627,7 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 	for _, file := range files {
 		fullPath := filepath.Join(resourcePath, file.Name())
 		obj, err := ctx.unmarshal(fullPath)
+		originalObj := obj.DeepCopy()
```

Comment:
Please move this down below the error check

Reply:
fixed